### PR TITLE
Updating dotnet sdk for NewRelic

### PR DESCRIPTION
**Pull request by vipray-ms**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by jsquire — DISMISSED** [review]
@vipray-ms:

Your account lacks the required public GitHub organizations and permissions required of an internal contributor.  Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

**Comment by vipray-ms** [comment]
> @vipray-ms:
> 
> Your account lacks the required public GitHub organizations and permissions required of an internal contributor. Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
> 
> * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> 
> You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.
> 
> Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

@jsquire 
- Joined Both Azure and Microsoft Organization

- Raised a request
image

- Added Codeowners

**Comment by jsquire** [comment]
@vipray-ms:  Your Azure and Microsoft organization memberships are not marked public, which is required.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -938,6 +938,13 @@
 # ServiceLabel: %Astronomer %Mgmt
 # ServiceOwners:                                                   @banggaurav
 
+# PRLabel: %NewRelicObservability
+/sdk/newrelicobservability/Azure.ResourceManager.NewRelicObservability/       @dipeshbhakat-microsoft @vipray-ms
```
Comment (jsquire) [inline .github/CODEOWNERS:942]:
```suggestion
/sdk/newrelicobservability/Azure.ResourceManager.*/                @dipeshbhakat-microsoft @vipray-ms
```

Comment (jsquire) [inline .github/CODEOWNERS:942]: dipeshbhakat-microsoft is missing the required public Azure org membership.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -938,6 +938,13 @@
 # ServiceLabel: %Astronomer %Mgmt
 # ServiceOwners:                                                   @banggaurav
 
+# PRLabel: %NewRelicObservability
```
Comment (jsquire) [inline .github/CODEOWNERS:941]: Let's change this to "New Relic"

**Comment by jsquire** [comment]
@vipray-ms : Please don't create arbitrary labels in the repository.  These have to be defined centrally and follow a consistent pattern to work with our automation and reporting.

File: .github/CODEOWNERS (hunk 2 above)
Comment (vipray-ms) [reply .github/CODEOWNERS]: Done

File: .github/CODEOWNERS (hunk 1 above)
Comment (vipray-ms) [reply .github/CODEOWNERS]: Done

Comment (vipray-ms) [reply .github/CODEOWNERS]: Done

**Comment by vipray-ms** [comment]
> @vipray-ms: Your Azure and Microsoft organization memberships are not marked public, which is required.

Done.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -938,6 +938,13 @@
 # ServiceLabel: %Astronomer %Mgmt
 # ServiceOwners:                                                   @banggaurav
 
+# PRLabel: %New Relic
+/sdk/newrelicobservability/Azure.ResourceManager.*/       @dipeshbhakat-microsoft @vipray-ms
+
+# ServiceLabel: %New Relic %Mgmt
+# ServiceOwners:                                                   @dipeshbhakat-microsoft @vipray-ms
+
+
```
Comment (jsquire) [inline .github/CODEOWNERS:947]:
```suggestion
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -938,6 +938,13 @@
 # ServiceLabel: %Astronomer %Mgmt
 # ServiceOwners:                                                   @banggaurav
 
+# PRLabel: %New Relic
```
Comment (jsquire) [inline .github/CODEOWNERS:941]: Please move this block down to ~L981 to preserve alphabetical ordering by label.

Comment (vipray-ms) [reply .github/CODEOWNERS]: Done

File: .github/CODEOWNERS (hunk 3 above)
Comment (vipray-ms) [reply .github/CODEOWNERS]: Done, Please release the latest version

**Comment by vipray-ms** [comment]
@ArthurMa1978, is anything else required from our side? Can you please let us know when can we expect the latest version?

**Comment by vipray-ms** [comment]
@ArthurMa1978 , Can you please provide us with the latest version of dot net SDK with this PR changes. 
CC: @dipeshbhakat-microsoft

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**